COMET.Web.Common.Test/COMET.Web.Common.Test.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+      <TargetFramework>net7.0</TargetFramework>
+      <LangVersion>Latest</LangVersion>
+      <ImplicitUsings>enable</ImplicitUsings>
+      <Version>1.0.0</Version>
+      <AssemblyVersion>1.0.0</AssemblyVersion>
+      <FileVersion>1.0.0</FileVersion>
+      <Title>COMET WEB Common Test</Title>
+      <Description>A Common Library that includes DevExpress Blazor and Tasks test helpers</Description>
+      <Company>RHEA System S.A.</Company>
+      <Copyright>Copyright 2023 RHEA System S.A.</Copyright>
+      <PackageId>COMET.WEB.Common.Test</PackageId>
+      <AssemblyTitle>COMET.WEB.Common.Test</AssemblyTitle>
+      <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+      <Authors>Sam, Alex, Alexander, Antoine</Authors>
```

Comment:
I assume that this library uses a lot of code done by other people, so add their names as well. As @nabilatrhea

Reply:
added you and nabil , well spotted !